src/main/java/core/basesyntax/FruitShopApplication.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import core.basesyntax.service.FruitShopController;
+
+public class FruitShopApplication {
+    public static void main(String[] args) {
+        FruitShopController controller = new FruitShopController();
+        controller.run();
+    }
```

Comment:
Make all services independent and call them in the right order in `main()` method step by step. The current implementation directly calls a controller which abstracts the flow of the program. Instead, instantiate and use the independent services here, like FileReader, DataConverter, OperationStrategy, etc., and orchestrate them to perform the program's operations.